README.md

```diff
@@ -65,17 +65,17 @@ Supported Compilers
+| TBB                                                                            | :white_check_mark:                              | :white_check_mark:                              | :white_check_mark:                          | :white_check_mark:     | :white_check_mark:                                         | :white_check_mark:                                    | :white_check_mark:                              | :white_check_mark:                          | :white_check_mark:                                | :white_check_mark:                        | :white_check_mark:                        | :white_check_mark:      | :white_check_mark:               | :white_check_mark:                                    | :white_check_mark:                   |
+| CUDA (nvcc)                                                                    | :white_check_mark: <br/> (CUDA 11.0 - 12.2)[^2] | :white_check_mark: <br/> (CUDA 11.4 - 12.0)[^2] | :white_check_mark: <br/> (CUDA 12.0 - 12.2) | :x:                    | :white_check_mark: <br/> (CUDA 11.0-11.2; 11.6 - 12.0)[^2] | :white_check_mark: <br/> (CUDA 11.2, 11.6 - 12.0)[^2] | :white_check_mark: <br/> (CUDA 11.6 - 12.0)[^2] | :white_check_mark: <br/> (CUDA 11.7 - 12.0) | :white_check_mark: <br/> (CUDA 11.8 - 12.0)       | :white_check_mark: <br/> (CUDA 12.2)      | :x:                                       | :x:                     | :x:                              | :x:                                                   | :x:                                  |
+| CUDA (clang)                                                                   | -                                               | -                                               | -                                           | :x:                    | :x:                                                        | :x:                                                   | :x:                                             | :x:                                         | :white_check_mark: (CUDA 11.0 - 11.5)             | :white_check_mark: (CUDA 11.0 - 11.5)[^1] | :white_check_mark: (CUDA 11.0 - 11.5)[^1] | :x:                     | :x:                              | -                                                     | -                                    |
```

Comment:
Why is CUDA (clang) not supported? Clang 17 supports up to CUDA 12.1: https://llvm.org/docs/CompileCudaWithLLVM.html
And we have a CI job, which test it.

Reply:
I added the range CUDA 11.0 - 12.1 then.

Reply:
According to https://github.com/llvm/llvm-project/blob/llvmorg-17.0.0/clang/include/clang/Basic/Cuda.h 11.8 is the last "fully supported" version, 12.1 is "partially supported". So far we always used the "fully supported" version in our matrix.

Reply:
If we want to stay consistent, yes.

Reply:
Alright.